### PR TITLE
Add online satbias capabilities

### PR DESCRIPTION
**Pull request by jak5808**
This commit adds online satbias data recycling, allowing users to grab satbias data from the previous cycle analysis instead of grabbing the data from GDAS each cycle.

### CODE REVIEW

**Review by evankalina — COMMENTED** [review]
Does this commit contain anything that updates the information in the satbias files from one cycle to the next, or does it simply carry those files over without modifying them? If the former, where is the update taking place? Thanks!

**Review by yonghuiweng — APPROVED** [review]

File: scripts/exhafs_analysis.sh
Comment (XuLu-NOAA) [inline scripts/exhafs_analysis.sh:449]: Hi, Bin, not sure if this is the same on Hera, but on Orion, these bias data is stored in ${COMgfs}/gdas.$PDYprior/${hhprior}/atmos/gdas.t${hhprior}z.abias for my Laura tests.
Both for analysis & enkf

**Comment by jak5808** [comment]
Hi @evankalina, the commit is intended to update the satbias files with each cycle. However upon further review today, I noticed a bug in which only the passive satbias files (satbias_pc) are being updated, and the assimilated satbias files remain the same. This is obviously an issue that I will be digging into ASAP.

Until we solve this problem, the online_satbias parameter should be set to "no" in hafs_basic.conf. Apologies for that mistake.

File: scripts/exhafs_analysis.sh
Comment (BinLiu-NOAA) [reply scripts/exhafs_analysis.sh:449]: @XL-OU This should be fine. The ${atmos} is "atmos/" or "" depending upon which version (GFSv15 or GFSV16) of the input GFS data is being used on all HAFS supported platforms.

**Comment by BinLiu-NOAA** [comment]
> Hi @evankalina, the commit is intended to update the satbias files with each cycle. However upon further review today, I noticed a bug in which only the passive satbias files (satbias_pc) are being updated, and the assimilated satbias files remain the same. This is obviously an issue that I will be digging into ASAP.
> 
> Until we solve this problem, the online_satbias parameter should be set to "no" in hafs_basic.conf. Apologies for that mistake.

@jak5808, No worries. Once you fixed the issue you can commit and push the fix in this branch. We will basically wait until you fix and test the fix to merge this PR.

File: scripts/exhafs_analysis.sh
Comment (XuLu-NOAA) [reply scripts/exhafs_analysis.sh:449]: Hi, @BinLiu-NOAA, I see.  It was my bad, somehow the atmos is not defined in my exchange_analysis.sh & exchange_enkf.sh.  But I found them in the most up-to-date versions.  Thx.

**Pull request closed without merging**
